.github/workflows/ci.yml

```
@@ -0,0 +1,50 @@
+name: 'Validate'
+
+on:
+  push:
+    branches:
+      - main
+      - polaris-release
+  pull_request:
+
+jobs:
+  build:
+    name: 'Build, Lint, & Test: node-${{ matrix.node_version }}'
```

```suggestion
    name: 'Build, Lint, and Test: node-${{ matrix.node_version }}'
```

In the spirit of calling out silly nits 🤓